.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: go
+go:
+  - "1.10"
+
+sudo: false
+
+before_script:
+  - go get -u -d github.com/lni/dragonboat
+  - cd $GOPATH/src/github.com/lni/dragonboat
+  - make install-rocksdb-ull
+  # - CGO_CFLAGS="-I/path/to/rocksdb/include" CGO_LDFLAGS="-L/path/to/rocksdb/lib -lrocksdb" go build -v pkgname
+  - go get github.com/lni/dragonboat-example/utils
+  - go get github.com/lni/dragonboat/config
+  - go get github.com/lni/dragonboat/logger
+  - go get github.com/lni/dragonboat/statemachine
+
+script:
+  # - make dragonboat-test
```

Comment:
It was too lengthy for Travis CI to support the verbose. Will fix this in the dragonboat & then perhaps enable it.